content/blog/2022-08-10-keda-2.8.0-release.md

```diff
@@ -0,0 +1,34 @@
++++
```

Comment:
Things to add:
- [ ] Activation and Scaling thresholds to delay activation until a certain threshold is hit (e.g. 100 unprocessed messages in a queue).
- [ ] min replica count for ScaledJobs
- [ ] Deprecation of `rolloutStrategy` in ScaledJob in favor of `rollout.strategy`
  - Should cover impact, when we remove it and how to migrate (latter is obvious)
- [ ] Customizing HPA name (optionally)

Reply:
The GitHub issue said activation threshold was in progress? Is it live in this release?

Reply:
@JorTurFer Are we done with that? I thought we were? I think we can close that issue.

Reply:
Merging as we can still pull it out afterwards